_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,286 @@
+---
+id: a25f45
+name: Headers attribute of td element refers to cells in the same table element
+rule_type: atomic
+description: |
+  This rule checks that the headers attribute of `td` elements refer to cells in the same `table` element that have a semantic role of `columnheader` or `rowheader`.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to any `td` element with a `headers` attribute, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
Should the rule not apply to the `headers` attribute instead of the `td` element?

Reply:
I did some testing on VoiceOver in general `headers` attributes seems to have a lot of inconsistencies. Also tested Firefox + nvda combo, with help of testing team.

Some findings:
- td scope=row does NOT work with VO or NVDA
- using headers="id1 id2" does NOT work 
- using headers to point to row headers in 2nd column does not work in VO, only works in NVDA/Firefox.
- `th` with `headers` attribute were also inconsistent.

Given the inconsistencies, I think it is best to scope this rule to `td` elements for this rule.

Reply:
All good then, though my question still stands: Should the test target be the `headers` attribute instead of the `td` element? This would be consistent with other rules that solely target attributes.